source/Octopus.Tentacle/Communications/HalibutInitializer.cs

```diff
@@ -100,8 +104,41 @@ void AddPollingEndpoints()
 
                 var halibutTimeoutsAndLimits = new HalibutTimeoutsAndLimits();
                 var serviceEndPoint = new ServiceEndPoint(pollingEndPoint.Address, pollingEndPoint.Thumbprint, halibutProxy, halibutTimeoutsAndLimits);
-                halibut.Poll(new Uri(pollingEndPoint.SubscriptionId), serviceEndPoint, CancellationToken.None);
+
+                var connectionCount = GetPollingConnectionCount();
+
+                for (var i = 0; i < connectionCount; i++)
+                {
+                    halibut.Poll(new Uri(pollingEndPoint.SubscriptionId), serviceEndPoint, CancellationToken.None);
+                }
+            }
+        }
+        
+        uint GetPollingConnectionCount()
+        {
+            //Open multiple polling connections if the env var is set to a non-zero/negative number
+            var connectionCount = 1u;
+            if (uint.TryParse(Environment.GetEnvironmentVariable(EnvironmentVariables.TentaclePollingConnectionCount), out var count))
+            {
+                log.InfoFormat("Requested polling connection count: {0}", count);
+                connectionCount = count;
             }
+
+            //Coerce the requested value as it might be outside our max & min
+            switch (connectionCount)
+            {
+                case > MaximumPollingConnectionCount:
+                    log.InfoFormat("Maximum polling connection count: {0}", MaximumPollingConnectionCount);
```

Comment:
Probably worth saying something like "The requested polling connection count is more than the maximum of 8, setting to 8 instead" etc.

Or just always print "Maximum allowed polling connection count" and the min allowed